helper/helper/src/main/java/com/linkedin/avroutil1/compatibility/exception/AvroUtilMissingFieldException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright 2022 LinkedIn Corp.
+ * Licensed under the BSD 2-Clause License (the "License").
+ * See License in the project root for license information.
+ */
+
+package com.linkedin.avroutil1.compatibility.exception;
+
+import java.util.ArrayList;
+import java.util.List;
+import org.apache.avro.AvroRuntimeException;
+import org.apache.avro.Schema;
+
+
+public class AvroUtilMissingFieldException extends AvroUtilException{
```

Comment:
Could we add to the javadoc as to how this exception is used? I think it would be useful for future readers (including ourselves :P ) when we are trying to figure out how it is used